src/components/inputs/CopyInput/CopyInput.tsx

```diff
@@ -66,26 +69,30 @@ export const CopyInput = (props: ICopyInputProps) => {
 			<div
 				className={classnames(styles.CopyInput, {
 					[styles.__Disabled]: disabled,
+					[styles.__Invalid]: isInvalid,
 				})}
 			>
-				<input
-					className={classnames(styles.CopyInput__Input, {
-						[styles.__Invalid]: isInvalid,
-					})}
-					id={styles.CopyInput__Input}
-					onChange={handleChange}
-					placeholder={placeholder}
-					type="text"
-					value={textToCopy}
-					disabled={disabled}
-					autoComplete="off"
-					spellCheck="false"
-				/>
+				<Tooltip
+					{...inputTooltipProps}
+					hideTooltip={inputTooltipProps ? inputTooltipProps.hideTooltip : true}
```

Comment:
Could you change the condition on the ternary to be `inputTooltipProps?.hideTooltip`? That way we know for sure that value is defined.